utils/src/server/rpc_module.rs

```diff
@@ -252,6 +260,51 @@ impl SubscriptionSink {
 		for entry in errored {
 			subs.remove(&entry);
 		}
+
 		Ok(())
 	}
+
+	/// Send a message to all subscriptions that could parse `P` as input.
+	///
+	/// F: is a closure that you need to provide to apply on the input P.
+	pub fn send_each<T, F>(&self, f: F) -> anyhow::Result<()>
+	where
+		F: Fn(&mut Params) -> T,
+		T: Serialize,
+	{
+		let mut subs = self.subscribers.lock();
+		let mut errored = Vec::new();
+
+		for ((conn_id, sub_id), sender) in subs.iter_mut() {
+			let result = match sender.params.as_mut().map(|p| to_raw_value(&f(p))) {
```

Comment:
```suggestion
			let result = match sender.params.as_ref().map(f) {
```
Closure return type is known, and it's ok if it's transient since it's going to be serialized in the same block, so there is no reason to use `RawValue` here.

Reply:
The reason was that [this](https://github.com/paritytech/jsonrpsee/blob/master/types/src/v2/params.rs#L17) requires RawValue

Reply:
You can just make the struct generic, no? :)

Reply:
yeah, that's possible but I'd prefer to do it another PR :)